src/tab.ts

```diff
@@ -62,7 +62,7 @@ export default class Tab extends LitElement {
     const hasChanged = isSelected !== this.#isSelected;
     this.#isSelected = isSelected;
 
-    if (isSelected && hasChanged) {
+    if (hasChanged) {
```

Comment:
Tab Group needs to know when a Tab has been deselected so it's not left without a selected Tab and can select the first Tab.